borgcron_all.sh

```diff
@@ -0,0 +1,177 @@
+#!/bin/sh
```

Comment:
This script seems to be a duplication of the original borgcron.sh file including whole backup logic. Code duplication is not good. I am also not sure how this works. The `backup_routine` function is not mentioned in this script.
I see I have to look at all files again…